### DISCUSSION TITLE
Sort Tags List (adding / editing tags)

### DISCUSSION THREAD

**Discussion by KangarooCreativeTeam** [jesec/flood#184] [Ideas]
Hi :D

The "set tags" list dropdown should be alphanumerically sorted (numbers first).  

(The "Untagged" option should always be first - this looks to already be implemented)

O_O

**Comment by jesec** [comment]
See: #135

Implemented by https://github.com/jesec/flood/commit/7d44626451db2d4825e1265392a45e23a7aa47d7.

**Comment by KangarooCreativeTeam** [comment]
O_O

Closing...
